packages/dashboard/package.json

```diff
@@ -120,6 +121,7 @@
     "react-hook-form": "^7.46.1",
     "react-hotkeys": "^2.0.0",
     "react-popper": "^2.3.0",
+    "react-resizable": "^3.0.5",
```

Comment:
should this on both devDependecy and dependency list?

Reply:
it is required for the functioning of the prop panel.. why dev ?

Reply:
the package.json change has it listed in both !

Reply:
oh no jk its the types up above, my bad cant read 🙈